### PR TITLE
docker-rename supported on docker containers

### PR DESCRIPTION
**Pull request by Yuki-Inoue**

### CODE REVIEW

**Comment by Silex** [comment]
Good idea. Can you add some `(error "foo")` when there is more than one container selected?

**Comment by Yuki-Inoue** [comment]
OK,, I'll make it act like the other popup commands.

**Comment by Yuki-Inoue** [comment]
@Silex done.

File: docker-containers.el [hunk 1]
```diff
@@ -156,6 +156,17 @@ Remove the volumes associated with the container when VOLUMES is set."
                                                             (s-join " " ,(list (intern (format "docker-containers-%s-arguments" it))))))
              functions)))
 
+;;;###autoload
+(defun docker-rename-entry ()
+  (interactive)
+  (docker-utils-select-if-empty)
+  (let ((ids (docker-utils-get-marked-items-ids)))
+    (if (<= 2 (length ids))
```
Comment (Silex) [inline docker-containers.el]: Surely that is wrong no? Pretty sure you want `(not (= 1 (length ids))`

**Comment by Silex** [comment]
Code review is actually quite enjoyable! :wink:

File: docker-containers.el (hunk 1 above)
Comment (Yuki-Inoue) [inline docker-containers.el]: You are so right.

**Comment by Yuki-Inoue** [comment]
@Silex done.

**Comment by Silex** [comment]
Thanks!

**Pull request merged**